### ISSUE TITLE
Several HTML tags have the same IDs

### ISSUE THREAD

**Issue by HrBjarup** [MalteMagnussen/CA2-FrontEnd#53]
There are several HTML tags on that have the same ID. I don't know if it's an actual problem right now but I still think it should be changed if we have time for it.

**Comment by Castau** [comment]
yes, in the navigation-html? I noticed that as well, but it seems to run fine anyway. I don't think they are used, but am not completely sure.
